pkg/cmd/container/list.go

```diff
@@ -100,7 +99,7 @@ func printContainers(ctx context.Context, client *containerd.Client, containers
 	case "raw":
 		return errors.New("unsupported format: \"raw\"")
 	case "wide":
-		w = tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
+		w = tabwriter.NewWriter(options.Stdout, 4, 8, 4, ' ', 0)
```

Comment:
could you please use the `w` var : `w = tabwriter.NewWriter(w, 4, 8, 4, ' ', 0)`